### PR TITLE
readme: add project name to readme

### PR DESCRIPTION
**Pull request by chrissexton**
I decided that the name of the project is Godot game.
This is a commit to test stuff.

Refs #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# project-coffee-capo-games
+# Godot Game
```
Comment (rngoode1) [inline README.md]: Wrong name. Correct name is Arachnivania.

Comment (chrissexton) [reply README.md]: OK, I'll push a new version

Comment (chrissexton) [reply README.md]: Done

**Review by rngoode1 — APPROVED** [review]
Looks good.

**Pull request merged**